src/main/kotlin/no/nav/syfo/behandler/partnerinfo/PartnerinfoClient.kt

```diff
@@ -59,7 +59,7 @@ class PartnerinfoClient(
                 COUNT_CALL_PARTNERINFO_MULTIPLE_RESPONSE.increment()
             }
 
-            return response.maxByOrNull { it.partnerId }
+            return response.minByOrNull { it.partnerId }
```

Comment:
Ser ut til at den del problemer blir forårsaket av duplikate partnerId'er i oppslag vi syfopartnerinfo. Kan se ut som det dukker opp nye feilaktige partnerId'er sporadisk, og hvis disse er større enn den opprinnelige vi hadde er det bedre å bruke minBy i stedet for maxBy.

Reply:
Har grav litt mer i dette, og tror ikke mønsteret er så klart - så reverterer denne endringen før merge.